propertyestimator/tests/test_utils_serialization.py

```diff
@@ -91,3 +92,13 @@ def test_polymorphic_dictionary():
     pydantic_recreated_json = pydantic_recreated.json()
 
     assert pydantic_json == pydantic_recreated_json
+
+
+def test_force_field_serialization():
+
+    from openforcefield.typing.engines import smirnoff
+
+    force_field = smirnoff.ForceField(get_data_filename('forcefield/smirnoff99Frosst.offxml'))
+
+    serialized_force_field = serialize_force_field(force_field)
+    deserialized_force_field = deserialize_force_field(serialized_force_field)
```

Comment:
It would be good to test equality for some deserialized force field parameters here. Maybe check that `force_field` and `deserialized_force_field` have the same number of parameters, and that the SMIRKS+k value for the first bond parameter is the same in both?

Reply:
Yeah I was trying to think if there are any good (read as 'simple') ways to do this that would give sufficient confidence that the two are actually the same!

Reply:
We do some counting tests in the `topology` branch. But I don't know how I'd write this in the old API. https://github.com/openforcefield/openforcefield/blob/topology/openforcefield/tests/test_forcefield.py#L158-L165